ansible/roles/kafka/tasks/deploy.yml

```diff
@@ -23,6 +23,7 @@
       "KAFKA_HEAP_OPTS": "-Xmx{{ kafka.heap }} -Xms{{ kafka.heap }}"
       "KAFKA_ZOOKEEPER_CONNECT": "{{ zookeeper_connect_string }}"
       "KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR": "{{ kafka.replicationFactor }}"
+      "KAFKA_AUTO_CREATE_TOPICS_ENABLE": "{{ kafka.autoCreateTopics }}"
```

Comment:
Honestly, I'd take the configuration out. We don't want auto-creation I think.

Reply:
That's fair. I'm removing the code to make this configurable and will simply switch automatic topic creation off. The default is `off` anyway and no Ansible environment has it `on`.